File: 01_Ruby_Small_Problems/Easy_4/short_long_short.rb
def short_long_short(string1, string2)
  if string1.length > string2.length
    string2.concat(string1, string2)
  else
    string1.concat(string2, string1)
  end
end

puts short_long_short('abc', 'defgh') == "abcdefghabc"
puts short_long_short('abcde', 'fgh') == "fghabcdefgh"
puts short_long_short('', 'xyz') == "xyz"

File: 06_JavaScript_Small_Problems/Easy_01/short_long_short.js
/*
- Take two strings as arguments
- determines the length of the two strings
- returns the results of concatenating short + long + short
*/

const shortLongShort = (str1, str2) => {
  if (str1.length < str2.length) {
    return (str1 + str2 + str1);
  } else if (str1.length > str2.length) {
    return (str2 + str1 + str2);
  }
};

console.log(shortLongShort('abc', 'defgh') === "abcdefghabc");
console.log(shortLongShort('abcde', 'fgh') === "fghabcdefgh");
console.log(shortLongShort('', 'xyz') === "xyz");